prov/verbs/src/verbs_info.c

```diff
@@ -982,6 +987,122 @@ static int vrb_ifa_rdma_info(const struct ifaddrs *ifa, char **dev_name,
 	return ret;
 }
 
+int vrb_get_port_space(const struct fi_info *info)
+{
+	if (info != NULL && info->addr_format == FI_SOCKADDR_IB)
+		return RDMA_PS_IB;
+	else
+		return RDMA_PS_TCP;
+}
+
+static int vrb_gid_is_null(const union ibv_gid *gid)
+{
+	return !(gid->raw[8]  | gid->raw[9]  | gid->raw[10] | gid->raw[11] |
+	         gid->raw[12] | gid->raw[13] | gid->raw[14] | gid->raw[15]);
+}
```

Comment:
This function is not checking for null gid.  It's checking that only half the git is 0

Reply:
Please see my comment below.